readerwriterqueue.h

```diff
@@ -237,6 +251,14 @@ class ReaderWriterQueue
 		return inner_enqueue<CanAlloc>(std::forward<T>(element));
 	}
 
+#if MOODYCAMEL_HAS_EMPLACE
+	// Like enqueue() but emplace semantics (ie construct-in-place)
```

Comment:
Nitpick: Missing with between but and emplace. Also, i.e. needs periods.

Reply:
Done.